### PR TITLE
add support command

### PR DESCRIPTION
**Pull request by Bishoy-at-pieces**
unnamed

it will automatically open the browser on the url if the user clicks on the option

### CODE REVIEW

File: misc/support_command.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import sublime
+import sublime_plugin
+import webbrowser
+
+
+support_resources = [
+	{
+		"text":"Open a github issue",
+		"value":"https://github.com/pieces-app/plugin_sublime/issues",
+		"details":"Create a github issue on github | https://github.com/pieces-app/plugin_sublime/issues"
```
Comment (mack-at-pieces) [inline misc/support_command.py:10]: "Create an issue on our GitHub"

File: misc/support_command.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+	},
+	{
+		"text":"Get support on our website",
```
Comment (mack-at-pieces) [inline misc/support_command.py:13]: "Get Support"

File: misc/support_command.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+		"value":"https://docs.pieces.app/support",
+		"details":"Checkout our website | https://docs.pieces.app/support"
+	},
+	{
+		"text":"Join our discord server",
```
Comment (mack-at-pieces) [inline misc/support_command.py:18]: "Join our Discord Server"

File: misc/support_command.py [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+support_resources = [
+	{
+		"text":"Open a github issue",
+		"value":"https://github.com/pieces-app/plugin_sublime/issues",
+		"details":"Create a github issue on github | https://github.com/pieces-app/plugin_sublime/issues"
+	},
+	{
+		"text":"Get support on our website",
+		"value":"https://docs.pieces.app/support",
+		"details":"Checkout our website | https://docs.pieces.app/support"
```
Comment (mack-at-pieces) [inline misc/support_command.py:15]: "Visit our website, https://docs.pieces.app/support"

File: misc/support_command.py [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+	},
+	{
+		"text":"Join our discord server",
+		"value":"https://discord.gg/getpieces",
+		"details":"Join our discord server | https://discord.gg/getpieces"
```
Comment (mack-at-pieces) [inline misc/support_command.py:20]: "Chat with the Pieces team and our community of developers around the world"

**Review by mack-at-pieces — CHANGES_REQUESTED** [review]

**Review by caleb-at-pieces — APPROVED** [review]

**Review by mark-at-pieces — APPROVED** [review]

**Review by mack-at-pieces — APPROVED** [review]

**Pull request merged**